Manually-importing-modpacks.md

```diff
@@ -0,0 +1,43 @@
+## Finding the pack location
+
+Install the pack of your choice per usual, then follow these steps to find its Minecraft folder.
+
+### Curseforge
+
+In the modpack overview, rightclick on the pack and click "Open Folder", keep this folder open we will need this for the last step.
+
+![](images/cf-openfolder.png)
+
+### FTB App
+
+Open the modpack in your library, click on "Settings" on the right side, then click the orange "Open Folder" button at the bottom, keep this folder open we will need this for the last step.
+
+![](images/ftbapp-openfolder.gif)
+
+## Creating the MultiMC instance
+
+First we will need to find the modloader version the pack uses, for launcher specific instructions see below. Once you have the packs modloader and version you can create [a Vanilla instance](Adding-an-Instance) using the packs Minecraft version. Then edit the instance, go to the [Version tab](Instance-Version) and click the respective "Install _modloader_" button in the left menu bar and install the packs modloader version.
+
+### Curseforge
+
+Open the modpack overview and the modloader is listed right of the game version
+
+![](images/cf-modloader-version.png)
+
+### FTB App
+
+The only way seems to be to read the latest changelog for the specific modpack, or check the version.json in the modpack folder. Just try the latest Forge/Fabric version for the packs Minecraft version, otherwise come ask for help in our Discord.
```

Comment:
instance.json

Reply:
cool, that's a lot more readable, updated!